### PR TITLE
Store location

### PR DESCRIPTION
**Pull request by dnpnghia**

### CODE REVIEW

**Review by vunn-2690 — APPROVED** [review]

**Comment by dnpnghia** [comment]
Ảnh chụp Màn hình 2021-10-04 lúc 14 34 24

**Comment by dnpnghia** [comment]
ready

**Comment by dnpnghia** [comment]
https://edu-redmine.sun-asterisk.vn/issues/42224

File: app/models/user.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class User < ApplicationRecord
   before_save :downcase_email
   enum status: {member: 0, admin: 1}
   has_many :orders, dependent: :destroy
+  has_one :shop, dependent: :destroy
```
Comment (vanvtt-0952) [inline app/models/user.rb:5]:
```suggestion
  has_one :shop, dependent: :destroy
  
```

**Comment by vanvtt-0952** [comment]
merged
comment nhỏ, fix kèm pull kế tiếp nhé bé

**Pull request merged**